cmd/analyzer.go

```diff
@@ -23,6 +25,10 @@ func (r *Analyzer) Run() (b *builder.Issues, err error) {
 	}
 	b = &builder.Issues{Path: output}
 	err = cmd.Run()
+	if err != nil {
```

Comment:
I would argue that something along this line would be useful, sounds like you have something in the works so will remove